README.md

```diff
@@ -248,138 +248,27 @@ This section is specifically geared towards deploying contracts on Avalanche Sub
-- Run through this tutorial to get all of the information you need about keys and deployment settings:
-   - [Deploy a Permissioned Subnet on Testnet](https://docs.avax.network/subnets/create-a-fuji-subnet#fuji-testnet)
-   - Finish up to **Add a Validator**
-- Next you will beed to become a validator:
-   - [Add Node as Validator on Fuji via the Web Wallet](https://docs.avax.network/nodes/build/launch-an-avalanche-validator-on-aws-with-one-click#add-node-as-validator-on-fuji-via-the-web-wallet)
-- Then, you'll need to add the validator:
-   - [Add a Validator](https://docs.avax.network/subnets/create-a-fuji-subnet#add-a-validator)
-   - The **Join a Subnet** section may seem like it is only talking about adding additional nodes, but that is not the case. **You do need to do it for your own current node as well.** So make sure you complete that step.
-- Finally, you'll need to update your subnet evm.
-  - Open up a session on your terminal
-  - Type: `systemctl stop avalanchego`, you will need to add `sudo` to the start of the command base on your session.
-  - To check if the service is off, type: `systemctl status avalanchego`.
-  - Install the subnet-evm VM to the avalanchego folder by following the step 1 from this tutorial online
-    - https://www.leewayhertz.com/create-a-custom-blockchain-on-avalanche/
-
-- Open up the terminal, manually navigate to the avalanchego folder and turn on the node using the command:
-  `cd ~/avalanche-node `
-  `avalanchego --network-id=fuji --track-subnets=<the newly created subnet-id>`
-- Then restart your node.
-- To verify your subnet has been deployed and is being validated you can navigate to this [website](https://testnet.avascan.info/blockchains) and search for your subnet ID.
-- The subnet for this tutorial can be found [here](https://testnet.avascan.info/blockchain/2PGZVnkoDTfV1ARcfdJB13B63PPYFsiav8UycsKByYhWoF79QT). You can see it has at least one validator.
+These instructions are for a Unix environment. If you have been doing this project in Windows up until this point, you'll need to make sure you set up your own WSL environment and then install the necessary prerequisites in the WSL environment as well. These are the install [instructions](https://learn.microsoft.com/en-us/windows/wsl/install) for WSL 1 as well as how to upgrade it to WSL 2. Additionally, Sequence only supports subnets for their paid developer tier, so you will need to make sure you are using Metamask.
 
+#### Use Kurtosis to create subnet and deploy a smart contract and turn up the game backend
+
+- Clone the project by running  
+  `git clone git@github.com:CodeLinkIO/MoveAndWhap.git`
+- Then run `cd MoveAndWhap`
+- To make sure nothing run yet, clear everything by running  
+  `kurtosis clean -a`
+- At the root folder, run  
+  `kurtosis run . '{"ephemeral_ports": true,"node_count": 1}' --enclave mnw`
```

Comment:
It will only works for Avalanche service, the Hardhat service due to network partition won't be able to connect to this Avalanche service if we set the `ephemeral_ports` to `false` because the RPC_URL returned from Avalanche service is now on the host machine's port which Hardhat service as a Docker container can't reach. It would be great for Kurtosis to provide an ability to run a service in the same network as the host network.